devnet/README.md

```diff
@@ -23,6 +23,39 @@ docker-compose down
 
 > **_NOTE:_**  The devnet is not stateful, so subsequent runs will start from the genesis block.
 
+## Running Cross Client Devnet
+
+A cross-client devnet is network of gossamer node(s) and substrate node(s).
+
+Steps to run a two node cross client devnet
+
+- Create `genesis-spec.json`
```

Comment:
Use numbers `1.` instead.

Tip 1: you need to indent the code blocks with 4 spaces so that the numbering is correct. Maybe that's why you were using bullet points instead 😉 

Tip 2: you can use `1.` for each step and markdown/github will auto-determine the right number in the enumeration, so you don't have to maintain `1.`, `2.`, etc.

Reply:
![download](https://user-images.githubusercontent.com/10185999/156399627-4713ae0b-1647-4622-a754-7cd758aa78d0.jpeg)